management/commands/aggregate_subject_trials.py

```diff
@@ -315,17 +418,110 @@ def handle(self, *_, **options):
 
         if dry:
             logger.info('Dry run complete: %s aggregate hash = %s', out_file, aggregate_hash)
-            return None, out_file, log_file
+            if compute_training_status:
+                _, training_out_file = self.handle_training_status(trials_table=out_file, rerun=rerun, dry=dry)
+                return None, [out_file, training_out_file], log_file
+            else:
+                return None, out_file, log_file
 
         # Create dataset
         dset, out_file = self.register_dataset(
-            out_file, file_hash=md5_hash, file_size=file_size, aggregate_hash=aggregate_hash, user=user, revision=revision)
+            out_file, file_hash=md5_hash, file_size=file_size, aggregate_hash=aggregate_hash, user=self.user,
+            revision=self.revision)
 
         # Move log file to new revision folder
         if out_file.parent != log_file.parent:
             log_file = log_file.rename(out_file.with_name(log_file.name))
 
-        return dset, out_file, log_file
+        if compute_training_status:
+            training_dset, training_out_file = self.handle_training_status(trials_table=out_file, rerun=rerun, dry=dry)
```

Comment:
Could `self.handle_training_status` be called once higher up, then taken out of the dry if-block?